source/MaterialXView/Editor.cpp

```diff
@@ -27,7 +29,7 @@ class EditorFormHelper : public ng::FormHelper
 
 PropertyEditor::PropertyEditor() :
     _visible(false),
-    _form(nullptr),
+    _container(nullptr),
```

Comment:
Replace usage of "form helper" and just keep a generic container and add in a scrolling pane + child widgets.